roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4583052&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4583052&tab=artifacts#/activerecord) on master @ [2f051c95ebb991e18134d2454807f992b05e4929](https://github.com/cockroachdb/cockroach/commits/2f051c95ebb991e18134d2454807f992b05e4929):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:866: 
		Tests run on Cockroach v22.1.0-alpha.3-269-g2f051c95eb
		Tests run against activerecord 6.1
		6649 Total Tests Run
		6647 tests passed
		2 tests failed
		19 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlIntervalTest#test_interval_type (unexpected)
		--- FAIL: BasicsTest#test_default_values_are_deeply_dupped (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #77322 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #69080 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-13839

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4632355&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4632355&tab=artifacts#/activerecord) on master @ [10e0c5d92f8ef953d6b497b448893bb5044cdd31](https://github.com/cockroachdb/cockroach/commits/10e0c5d92f8ef953d6b497b448893bb5044cdd31):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:875: 
		Tests run on Cockroach v22.1.0-alpha.4-169-g10e0c5d92f
		Tests run against activerecord 6.1
		6649 Total Tests Run
		6648 tests passed
		1 test failed
		19 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #78120 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.1 release-blocker]
- #77322 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #69080 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4777887&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4777887&tab=artifacts#/activerecord) on master @ [9a66e9ad26bc34250bee89854cbce1999ea70f9f](https://github.com/cockroachdb/cockroach/commits/9a66e9ad26bc34250bee89854cbce1999ea70f9f):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:875: 
		Tests run on Cockroach v22.1.0-alpha.4-610-g9a66e9ad26
		Tests run against activerecord 6.1
		6649 Total Tests Run
		6648 tests passed
		1 test failed
		19 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #79204 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4831888&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4831888&tab=artifacts#/activerecord) on master @ [7a97bdd02c62787f40d79d5a2cd3439d8c9c8a1b](https://github.com/cockroachdb/cockroach/commits/7a97bdd02c62787f40d79d5a2cd3439d8c9c8a1b):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:875: 
		Tests run on Cockroach v22.1.0-alpha.4-810-g7a97bdd02c
		Tests run against activerecord 6.1
		6649 Total Tests Run
		6648 tests passed
		1 test failed
		19 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: BasicsTest#test_default_values_are_deeply_dupped (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #79491 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.1]
- #79204 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4844086&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4844086&tab=artifacts#/activerecord) on master @ [ea4c75e28696ada91135346eade10366b1642ae4](https://github.com/cockroachdb/cockroach/commits/ea4c75e28696ada91135346eade10366b1642ae4):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:190,orm_helpers.go:116,activerecord.go:251,test_runner.go:875: 
		Tests run on Cockroach v22.1.0-alpha.4-844-gea4c75e286
		Tests run against activerecord 6.1
		6649 Total Tests Run
		6648 tests passed
		1 test failed
		19 tests skipped
		3 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_1) is available in the artifacts' activerecord log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #79646 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #79491 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.1]
- #79204 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/activerecord.go
+++ b/pkg/cmd/roachtest/tests/activerecord.go
@@ -27,7 +27,7 @@ import (
 
 var activerecordResultRegex = regexp.MustCompile(`^(?P<test>[^\s]+#[^\s]+) = (?P<timing>\d+\.\d+ s) = (?P<result>.)$`)
 var railsReleaseTagRegex = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)\.?(?P<subpoint>\d*)$`)
-var supportedRailsVersion = "6.1"
+var supportedRailsVersion = "6.1.5"
 var activerecordAdapterVersion = "v6.1.8"
 
 // This test runs activerecord's full test suite against a single cockroach node.

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -37,6 +37,7 @@ var activeRecordBlockList21_1 = blocklist{}
 var activeRecordBlockList20_2 = blocklist{}
 
 var activeRecordIgnoreList22_1 = blocklist{
+	"CockroachDB::PostgresqlIntervalTest#test_interval_type":               "flaky",
 	"ConcurrentTransactionTest#test_transaction_isolation__read_committed": "flaky - https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/237",
 	"FixturesTest#test_create_fixtures":                                    "flaky - FK constraint violated sometimes when loading all fixture data",
 	"IgnoreFixturesTest#test_ignores_books_fixtures":                       "flaky - FK constraint violated sometimes when loading all fixture data",
@@ -45,6 +46,7 @@ var activeRecordIgnoreList22_1 = blocklist{
 
 var activeRecordIgnoreList21_2 = blocklist{
 	"ActiveRecord::CockroachDB::UnloggedTablesTest#test_gracefully_handles_temporary_tables": "modified to pass on 20.2",
+	"CockroachDB::PostgresqlIntervalTest#test_interval_type":                                 "flaky",
 	"ConcurrentTransactionTest#test_transaction_isolation__read_committed":                   "flaky - https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/237",
 	"FixturesTest#test_create_fixtures":                                                      "flaky - FK constraint violated sometimes when loading all fixture data",
 	"IgnoreFixturesTest#test_ignores_books_fixtures":                                         "flaky - FK constraint violated sometimes when loading all fixture data",
@@ -53,9 +55,10 @@ var activeRecordIgnoreList21_2 = blocklist{
 
 var activeRecordIgnoreList21_1 = blocklist{
 	"ActiveRecord::CockroachDB::UnloggedTablesTest#test_gracefully_handles_temporary_tables": "modified to pass on 20.2",
-	"FixturesTest#test_create_fixtures":                "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_books_fixtures":   "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_parrots_fixtures": "flaky - FK constraint violated sometimes when loading all fixture data",
+	"CockroachDB::PostgresqlIntervalTest#test_interval_type":                                 "flaky",
+	"FixturesTest#test_create_fixtures":                                                      "flaky - FK constraint violated sometimes when loading all fixture data",
+	"IgnoreFixturesTest#test_ignores_books_fixtures":                                         "flaky - FK constraint violated sometimes when loading all fixture data",
+	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                                       "flaky - FK constraint violated sometimes when loading all fixture data",
 }
 
 var activeRecordIgnoreList20_2 = blocklist{

